srcCpp03/RTIDDSImpl.cxx

```diff
@@ -80,10 +80,11 @@ int DynamicDataMembersId::at(std::string key)
 }
 
 template <typename T>
-RTIDDSImpl<T>::RTIDDSImpl():
+RTIDDSImpl<T>::RTIDDSImpl(bool isFlatData):
         _InstanceMaxCountReader(dds::core::LENGTH_UNLIMITED), //(-1)
```

Comment:
why is `//(-1)` a relevant comment?

Reply:
I do not know, it was there before I committed my changes. I guess it is to clarify what is the actual value of `dds::core::LENGTH_UNLIMITED` (which is -1).

Nevertheless, I think `dds::core::LENGTH_UNLIMITED` is self-explanatory and therefore `(-1)` should be removed.

I will wait until @javmorcas gives feedback about it

Reply:
I agree with you 2.